woocommerce/woocommerce
# make `wccom` commands text translatable, add application-password section URL

### All Submissions:






### Changes proposed in this Pull Request:



Continues #34073.

### How to test the changes in this Pull Request:

1. Follow instructions from #34073
2. Run `wp wc com connect --force` to force URL being displayed
3. Confirms command works as expected
<img width="1278" alt="Screen Shot 2022-08-17 at 13 44 53" src="https://user-images.githubusercontent.com/532402/185228939-adfe6a17-e4aa-48c3-aaab-3d15f5c9ae80.png">
